### ISSUE TITLE
Improved hyperparameter sweep UI

### ISSUE THREAD

**Issue by Danmou** [wandb/client#598]
`wandb --version && python --version && uname`

* Weights and Biases version: 0.8.12
* Python version: 3.6.9
* Operating System: Debian 10

### Description
Currently when you open a sweep in W&B, no useful visualizations are created by default. You have to manually create plots to visualize what's going on, and the parameters that you are actually varying are not suggested first so this is a rather tedious process. Furthermore, plots in the auto visualizations are labeled with the randomly generated run names, making it hard to figure out which set of HPs correspond to which line. It would be really nice to have an interface like TensorBoard 2.0's HParams (try it in Colab [here](https://colab.research.google.com/github/tensorflow/tensorboard/blob/master/docs/hyperparameter_tuning_with_hparams.ipynb#scrollTo=oxrSUAnCeFmQ&line=2&uniqifier=1))
![](https://raw.githubusercontent.com/tensorflow/tensorboard/master/docs/images/hparams_parallel_coordinates.png)

With this interface, a parallel coordinates plot and a grid of scatter plots are automatically created based on the metric(s) and hyperparameters, each of which can be easily toggled on or off. Mouseover on the PC plot highlights the line and shows the associated HPs and metrics at the bottom. The PC plot also supports log and quantile scales for individual axes, which is not currently supported by W&B.

An additional visualization that would be nice is something like [plot_objective from scikit-optimize](https://scikit-optimize.github.io/plots.m.html#skopt.plots.plot_objective):
![](https://miro.medium.com/max/3200/1*z2cufr9Y0OAkCFPLKWaFbg.png)

Another part that could use some improvement is the sweeps overview. It would be nice if you could rename sweeps, see which parameters were sweeped, see best metrics, delete sweeps etc. directly from the overview. A way to visualize multiple sweeps, or even any selection of runs, jointly with the above described interface would also be useful.

Anyway thanks for creating a really useful tool, and I hope with these improvements it will become even better!

**Comment by cvphelps** [comment]
Thanks for these ideas and suggestions! I'll share them with the team.

**Comment by raubitsj** [comment]
We have added auto visualizations for sweeps in the ui over the past couple of weeks!  Please give it a try and let us know what you think.

You can rename and delete sweeps also.

**Comment by ariG23498** [comment]
Hey @Danmou 
In the past year we've majorly reworked the CLI and UI for Weights & Biases. We're closing issues older than 6 months. Please comment to reopen.
